src/index.html

```diff
@@ -11,7 +11,144 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <main>
+      <form
+        action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+        method="post"
+      >
+        <fieldset>
+          <legend>Personal information</legend>
+          <div class="form input">
+            <label for="surname">Surname:</label>
+            <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+            >
```

Comment:
check your code style according to checklist
<img width="919" alt="image" src="https://user-images.githubusercontent.com/102411130/225564579-3e355eae-3cc6-46ae-8416-fbb9ca1a5b05.png">

Reply:
fix all such cases

Reply:
Is it allowed to use Prettier for fixing such cases?